### ISSUE TITLE
Can't delete an alias when an alias was added with just one email.

### ISSUE THREAD

**Issue by mcastillof** [tomav/docker-mailserver#1563]
## Context
While using ./setup.sh, If you create an alias with an uncompleted command, then you can't delete it.

##Steps to reproduce:
```
user@zeus:~/mail# ./setup.sh alias list
alias1@domain.com account1@domain.com
user@zeus:~/mail# ./setup.sh alias add alias2@domain.com
user@zeus:~/mail# ./setup.sh alias list
alias1@domain.com account1@domain.com
alias2@domain.com
user@zeus:~/mail# ./setup.sh alias del alias2@domain.com
Usage: delalias <alias@domain> <recipient@other>
No email specifed
```

## Actual Behavior
The malformed alias can't be deleted

## Expected behavior (i.e. solution)
The malformed alias be deleted 

## Your Environment
'latest' docker image with latest ./setup.sh file.

## Workaround
I deleted the alias entry in /tmp/docker-mailserver/postfix-virtual.cf inside the container.

## About a real solution

Both addalias and delalias expects two arguments:
```
/usr/local/bin/addalias
Usage: addalias <alias@domain> <recipient@other>
```

```
/usr/local/bin/delalias
Usage: delalias <alias@domain> <recipient@other>
```

but, addalias just check for one argument. 
https://github.com/tomav/docker-mailserver/blob/df4e04f03346e8f384e6857f0d8240511c2ed63d/target/bin/addalias#L21
https://github.com/tomav/docker-mailserver/blob/df4e04f03346e8f384e6857f0d8240511c2ed63d/target/bin/delalias#L21

I'm not an email expert, so I don't know if a line with just one email is valid in the postfix-virtual.cf file. if it is valid, then delalias should be modified to be able to delete a line with one email. If it is not valid, then addalias should be modified to ensure that two arguments are there.

**Comment by erik-wramner** [comment]
You're absolutely right. Feel free to make a PR. Fortunately you can just edit the file directly (as you did), so at least there is a good workaround.

**Comment by mcastillof** [comment]
@erik-wramner So, should I change addalias to ensure that both arguments are there, or should I change delalias to allow operation with one argument? I'm not a mail config expert, I have been learning here with this repo.

**Comment by casperklein** [comment]
>So, should I change addalias to ensure that both arguments are there

Yes, the second argument is mandatory.

Further information: http://www.postfix.org/VIRTUAL_README.html

**Comment by erik-wramner** [comment]
Sorry for the slow answer. The addalias command needs both arguments, but AFAIK delalias should work with only one argument. The PR (and indeed the old code) insists on two arguments for delalias.

**Comment by aendeavor** [comment]
I just implemented the fix, which will be merged with #1601. @erik-wramner You need the second argument, even for `./setup.sh alias del <...> <...>`.

**Comment by erik-wramner** [comment]
Yes I noticed a bit later.
